plugin.audio.criminalpodcast/addon.py

```diff
@@ -0,0 +1,75 @@
+from xbmcswift2 import Plugin, xbmcgui
```

Comment:
This module is not python3 compatible which makes your addon also not python3 compatible. From leia onwards we require all addons (submitted to branches higher or equal to leia) to be python3 compatible.